apps/web/src/pages/settings/components/ApiKeysCard.tsx

```diff
@@ -2,32 +2,69 @@ import React from 'react';
 import { ActionIcon } from '@mantine/core';
 import { useClipboard } from '@mantine/hooks';
 import { useQuery } from 'react-query';
-import { Input, Tooltip } from '../../../design-system';
+import { Input, Text, Tooltip, colors } from '../../../design-system';
 import { Check, Copy } from '../../../design-system/icons';
-import Card from '../../../components/layout/components/Card';
 import { getApiKeys } from '../../../api/environment';
+import { useEnvironment } from '../../../api/hooks/use-environment';
+import styled from 'styled-components';
 
 export const ApiKeysCard = () => {
   const clipboard = useClipboard({ timeout: 1000 });
-  const { data: apiKeys, isLoading: isLoadingApiKeys } = useQuery<{ key: string }[]>('getApiKeys', getApiKeys);
+  const { data: apiKeys } = useQuery<{ key: string }[]>('getApiKeys', getApiKeys);
+  const { environment } = useEnvironment();
+
+  const apiKey = apiKeys?.length ? apiKeys[0].key : '';
+  const environmentIdentifier = environment?.identifier ? environment.identifier : '';
 
   return (
-    <Card title="Api Keys">
-      <div style={{ maxWidth: '600px' }}>
+    <>
+      <ParamContainer>
+        <SmallTitle weight="bold">Api Key</SmallTitle>
+        <Description>Use this api key to interact with the novu api</Description>
         <Input
-          label="Use this api key to interact with the novu api"
           readOnly
+          type={'password'}
           rightSection={
-            <Tooltip label={clipboard.copied ? 'Copied!' : 'Copy Key'}>
-              <ActionIcon variant="transparent" onClick={() => clipboard.copy(apiKeys?.length ? apiKeys[0].key : '')}>
+            <Tooltip id-test={'Tooltip'} label={clipboard.copied ? 'Copied!' : 'Copy Key'}>
+              <ActionIcon variant="transparent" onClick={() => clipboard.copy(apiKey)}>
                 {clipboard.copied ? <Check /> : <Copy />}
               </ActionIcon>
             </Tooltip>
           }
-          value={apiKeys?.length ? apiKeys[0].key : ''}
+          value={apiKey}
           data-test-id="api-key-container"
         />
-      </div>
-    </Card>
+      </ParamContainer>
+      <ParamContainer>
+        <SmallTitle weight="bold">Application Identifier</SmallTitle>
+        <Description>A public key identifier that can be exposed to the client applications</Description>
+        <Input
```

Comment:
@djabarovgeorge Is it possible to use here the `InputWrapper` aswell? instead of re creating it here?

Reply:
Let me look into what i can do :)